### PR TITLE
Converter for BE-VLG

### PR DESCRIPTION
**Pull request by ivorbosloper**
A Belgium-Flanders converter. The official downloadable package `file_GPKG.zip` doesn't contain a file `file_GPKG.gpkg`, so the file needs to be opened as `/vsizip/file_GPKG.zip/real_file_name.gpkg` . In order to keep both file-caching and `/vsizip/` compatibility, I introduced a `path_template` parameter. Didn't test this on windows paths...

### CODE REVIEW

File: fiboa_cli/datasets/be_vlg.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+from ..convert_utils import convert as convert_
+
+URI = "https://www.landbouwvlaanderen.be/bestanden/gis/Landbouwgebruikspercelen_2023_-_Definitief_(extractie_28-03-2024)_GPKG.zip"
+# in the zipfile, the gpkg has a different name so we need to add this while opening the file
+PATH_TEMPLATE = "/vsizip/{}/Landbouwgebruikspercelen_2023_-_Definitief_(extractie_28-03-2024).gpkg"
```
Comment (m-mohr) [inline fiboa_cli/datasets/be_vlg.py:5]: That's an interesting idea, let me test this on Windows and with some other converters where we need this, too (and solved it with just extracting from ZIP manually), e.g. at and the eurocrops datasets.

**Review by m-mohr — CHANGES_REQUESTED** [review]
Thank you! Looks very good, just a clarification on the license needed, fixing the attribution and then checking the path_template on my side. I may extract the path_template into a separate commit/PR.

File: fiboa_cli/datasets/be_vlg.py (hunk 1 above)
Comment (m-mohr) [reply fiboa_cli/datasets/be_vlg.py:5]: I ran `fiboa convert be_vlg -o be -collection -c be.gpkg` and it returned

> '/vsizip/be.gpkg/Landbouwgebruikspercelen_2023_-_Definitief_(extractie_28-03-2024).gpkg' does not exist in the file system, and is not recognized as a supported dataset name. I guess this doesn't quite work with the -c option yet?

File: fiboa_cli/datasets/be_vlg.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+From 2023, the downloadable dataset of agricultural use plots will also include the specialization given by the company (= company typology) and that is given to the plots of the company. Based on the typology, the companies are divided into 4 major specializations: arable farming, horticulture, livestock farming and mixed farms. The specialization of each company is calculated annually according to a European method and is based on the standard output of the various agricultural productions on the company. It is therefore an economic specialization and not a reflection of all agricultural production on the company.
+"""
+BBOX = [2.531029700947351, 50.67422857041101, 5.932736829503416, 51.49545715218195]
+
+PROVIDER_NAME = "Agentschap Landbouw & Zeevisserij (Government)"
+PROVIDER_URL = "https://landbouwcijfers.vlaanderen.be/open-geodata-landbouwgebruikspercelen"
+
+ATTRIBUTION = ""
+LICENSE = "CC0"  # Publiek / Toegang zonder voorwaarden
```
[the remark this answers is not in the document]
Comment (m-mohr) [reply fiboa_cli/datasets/be_vlg.py:20]: If it is, then it should be:

```suggestion
LICENSE = "CC0-1.0"  # Publiek / Toegang zonder voorwaarden
```

File: fiboa_cli/datasets/be_vlg.py (hunk 1 above)
Comment (m-mohr) [reply fiboa_cli/datasets/be_vlg.py]: To unblock this PR I've changed the code to use the same manual unzipping approach as in the other converters for now. I'll follow-up with a better solution through issue #47.

File: fiboa_cli/datasets/be_vlg.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ivorbosloper) [reply fiboa_cli/datasets/be_vlg.py]: On a closer look, the license appeared to be a custom license, compatible with CC-BY-3.0. I simplified it by using `LICENSE="CC-BY-3.0"`. Please check if you agree...

[the remark this answers is not in the document]
Comment (m-mohr) [reply fiboa_cli/datasets/be_vlg.py]: Hmm, compatible doesn't mean it's licensed as such. This should probably be a link then.

File: fiboa_cli/datasets/be_vlg.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+- https://metadata.vlaanderen.be/srv/dut/catalog.search#/metadata/ae94435d-1ffb-451b-9c8b-d7e8c2d42dc2
+- https://metadata.vlaanderen.be/srv/dut/catalog.search#/metadata/21dda3b2-6e27-4bf3-af34-013854331f41
+
+Bronvermeldingsvoorschrift (Attribution requirement): "Bron: Dept. LV"
+"""
+
+ATTRIBUTION = "Bron: Dept. LV"
+# Compatible with CC-BY-3.0, license https://data.vlaanderen.be/id/licentie/modellicentie-gratis-hergebruik/v1.0
+LICENSE = "CC-BY-3.0"
```
Comment (m-mohr) [inline fiboa_cli/datasets/be_vlg.py:37]:
```suggestion
LICENSE = {"title": "Licentie modellicentie-gratis-hergebruik/v1.0", "href": "https://data.vlaanderen.be/id/licentie/modellicentie-gratis-hergebruik/v1.0", "type": "text/html", "rel": "license"}
```

File: fiboa_cli/datasets/be_vlg.py [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+This documentation states the license is compatible with Creative Commons Attribution license 3.0 (CC BY 3.0)
+
+- https://metadata.vlaanderen.be/srv/dut/catalog.search#/metadata/ae94435d-1ffb-451b-9c8b-d7e8c2d42dc2
+- https://metadata.vlaanderen.be/srv/dut/catalog.search#/metadata/21dda3b2-6e27-4bf3-af34-013854331f41
+
+Bronvermeldingsvoorschrift (Attribution requirement): "Bron: Dept. LV"
+"""
+
+ATTRIBUTION = "Bron: Dept. LV"
+# Compatible with CC-BY-3.0, license https://data.vlaanderen.be/id/licentie/modellicentie-gratis-hergebruik/v1.0
```
Comment (m-mohr) [inline fiboa_cli/datasets/be_vlg.py:36]:
```suggestion
# License is https://data.vlaanderen.be/id/licentie/modellicentie-gratis-hergebruik/v1.0, which is compatible with CC-BY-3.0
```

**Review by m-mohr — APPROVED** [review]

**Pull request merged**